versions/3.0.4.md

```diff
@@ -182,6 +182,8 @@ Models are defined using the [Schema Object](#schemaObject), which is an extende
 OAS uses several known formats to define in fine detail the data type being used.
 However, to support documentation needs, the `format` property is an open `string`-valued property, and can have any value.
 Formats such as `"email"`, `"uuid"`, and so on, MAY be used even though they are not defined by this specification.
+The OpenAPI Initiative also hosts a [Format Registry](https://spec.openapis.org/registry/format/) for formats defined by OAS users and other specifications.  Support for any registry format is strictly OPTIONAL, and support for one registry format does not imply support for any others.
```

Comment:
I don't understand

> and support for one registry format does not imply support for any others

and would be less confused with ending the sentence after "OPTIONAL".

Reply:
How about "registered format"? I was mislead to think about "format for the registry itself and its entries".

```suggestion
The OpenAPI Initiative also hosts a [Format Registry](https://spec.openapis.org/registry/format/) for formats defined by OAS users and other specifications.  Support for any registered format is strictly OPTIONAL, and support for one registered format does not imply support for any others.
```